### DISCUSSION TITLE
Use script to set up auth token

### DISCUSSION THREAD

**Discussion by yev-stash** [riok/Kreya#217] [General]
hey guys!
In my project I have a login call which returns me token in response. And then I need to sign every call with received token. Is there any way I can update value of the static value typed authentication by script after login call? 

Current implementation is not working because it seems kreya reads variables once at the moment I am updating Auths. 

<img width="928" alt="image" src="https://github.com/riok/Kreya/assets/25328829/bc191da3-d21e-41c8-9030-4fbcd590a8e1">

**Comment by latonz** [comment]
Thank you for reaching out.
I cannot reproduce this myself with Kreya `1.13.1`. We fixed exactly this with [Kreya 1.13.0](https://kreya.app/docs/release-notes/#113020240129).

If you are using an outdated version of Kreya could you try the latest version (you can check the version and update Kreya in the about window)? The latest version currently is `1.13.1`.

If it doesnt work with `1.13.1`, can you share the script you are using to set `vars.token`? Are you using directory settings or project default settings to set the `Token Local` value?

**Comment by yev-stash** [comment]
hey @latonz, thank you so much for your help. It was not latest version of kreya on my side. Now it works. I will share scripts maybe it will help somebody to fulfill same logic...
 - script for setting token variable
 
<img width="548" alt="image" src="https://github.com/riok/Kreya/assets/25328829/f437c261-4612-4907-b23c-5765ac94305f">
 - response 
 
<img width="306" alt="image" src="https://github.com/riok/Kreya/assets/25328829/a0427cab-a444-41d3-aeee-0c7993f327f5">

**Reply by latonz** [reply]
No worries, I‘m glad it works now 😊
Are you using a package manager or a similar tool to manage your Kreya installation? Usually Kreya should update itself automatically, I‘m wondering why this didn‘t happen here…
